pkg/installer/endpoints.go

```diff
@@ -137,3 +205,47 @@ func etcdctlHealthCheck(config *rest.Config, etcdShellPodName, etcdShellPodNS, e
 
 	return nil
 }
+
+func etcdShellPodTLS() string {
```

Comment:
What is the reason of a function instead of constant?

Reply:
no good reason 😄 changed to const